app/ts/utils/ethereumByteCodes.ts

```diff
@@ -59,3 +59,27 @@ contract timeLockMulticall {
 }
+
+/*
+library GetCode {
+	function at(address addr) public view returns (bytes memory code) {
+		assembly {
+			// retrieve the size of the code, this needs assembly
+			let size := extcodesize(addr)
+			// allocate output byte array - this could also be done without assembly
+			// by using code = new bytes(size)
+			code := mload(0x40)
+			// new "memory end" including padding
+			mstore(0x40, add(code, and(add(add(size, 0x20), 0x1f), not(0x1f))))
+			// store length in memory
+			mstore(code, size)
+			// actually retrieve the code, this needs assembly
+			extcodecopy(addr, add(code, 0x20), 0, size)
+		}
+	}
+}
+*/
+
+export const getCodeByteCode = () => {
+	return EthereumData.parse('0x73000000000000000000000000000000000000000030146080604052600436106100355760003560e01c8063dce4a4471461003a575b600080fd5b610054600480360381019061004f91906100f8565b61006a565b60405161006191906101b5565b60405180910390f35b6060813b6040519150601f19601f602083010116820160405280825280600060208401853c50919050565b600080fd5b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b60006100c58261009a565b9050919050565b6100d5816100ba565b81146100e057600080fd5b50565b6000813590506100f2816100cc565b92915050565b60006020828403121561010e5761010d610095565b5b600061011c848285016100e3565b91505092915050565b600081519050919050565b600082825260208201905092915050565b60005b8381101561015f578082015181840152602081019050610144565b60008484015250505050565b6000601f19601f8301169050919050565b600061018782610125565b6101918185610130565b93506101a1818560208601610141565b6101aa8161016b565b840191505092915050565b600060208201905081810360008301526101cf818461017c565b90509291505056fea26469706673582212206a5f0cd9f230619fa520fc4b9d4b518643258cad412f2fa33945ce528b4b895164736f6c63430008120033')
```

Comment:
Can we have this bytecode generated by the build process rather than hard coded?  Would make me much more comfortable including it, and make it far easier to audit.

Reply:
Same for the bytecode above.